egs2/l3das22/enh1/local/data.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+
+help_message=$(cat << EOF
+Usage: $0 
+  optional argument:
+    None
+EOF
+)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+
+stage=1
+stop_stage=2
+
+. utils/parse_options.sh
+
+if [ $# -ne 0 ]; then
+    echo "${help_message}"
+    exit 1;
+fi
+
+if [ ! -e "${L3DAS22}" ] ; then
+    log "
+    Please fill the value of 'L3DAS22' in db.sh
+    The 'L3DAS22' (https://www.kaggle.com/l3dasteam/l3das22) directory 
+    should at least contain the task1 dataset:
```

Comment:
In that case, does it possible to directly download the data from kaggle API?

Reply:
Added downloading the data from kaggle.